src/components/messenger/autocomplete-members/index.tsx

```diff
@@ -65,12 +64,6 @@ export class AutocompleteMembers extends React.Component<Properties, State> {
           inputClassName={'autocomplete-members__search-input'}
         />
 
-        {this.state.results?.length === 0 && this.state.results !== '' && (
-          <div className={classNames('messages-list__empty', 'messages-list__empty-top-padding')}>
```

Comment:
do we need the `messages-list__empty` in the styles now?

Reply:
Great spot! Thanks @ratik21